MANIFEST.in

```diff
@@ -1,3 +1,2 @@
-include pyproject.toml
-include *.md
-include LICENSE
+include CONTRIBUTING.md Makefile bin/*.py requirements/*
```

Comment:
Break these into separate lines for readability. Use recursive-include for `bin` and `requirements`.